icu4c/source/i18n/unicode/uspoof.h

```diff
@@ -359,6 +359,45 @@
  * COMMON or INHERITED, such as numbers and punctuation, are ignored when computing whether a string has multiple
  * scripts.
  *
+ * <h2>Advanced bidirectional usage</h2>
+ * If the paragraph direction with which the identifiers will be displayed is not known, there are
+ * multiple options for confusable detection depending on the circumstances.
+ *
+ * <p>
+ * In some circumstances, the only concern is confusion between identifiers displayed with the same
+ * paragraph direction.  For instance, if the identifiers are usernames prefixed with the @ symbol,
```

Comment:
FYI/no need to change: I am personally not a fan of the double space after a sentence. Looks weird, looks like a typo, and makes me stumble every time. :-)

Reply:
https://xkcd.com/1285/
For some reason I only do this in code comments, never elsewhere. Well, I know the reason, @pleroy taught me that when reviewing my code :-p